shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-mysql/src/main/java/org/apache/shardingsphere/proxy/frontend/mysql/err/MySQLErrPacketFactory.java

```diff
@@ -116,6 +116,9 @@ public static MySQLErrPacket newInstance(final Exception cause) {
         if (cause instanceof ScalingJobNotFoundException) {
             return new MySQLErrPacket(1, CommonErrorCode.SCALING_JOB_NOT_EXIST, ((ScalingJobNotFoundException) cause).getJobId());
         }
+        if (cause instanceof RuntimeException) {
+            return new MySQLErrPacket(1, CommonErrorCode.RUNTIME_EXCEPTION, cause.getMessage());
+        }
         return new MySQLErrPacket(1, CommonErrorCode.UNKNOWN_EXCEPTION, cause.getMessage());
```

Comment:
What's the main difference betweenen there two exception？

Reply:
There is no difference between the two, except that the display on the client side has changed from `unknown exception` to `runtime exception`.
It mainly solves the `unknown exception` that appeared in the last poc.